.github/workflows/ci.yml

```diff
@@ -17,7 +17,10 @@ jobs:
           node-version: ${{ matrix.node-version }}
           cache: "npm"
       - run: npm ci
+      - run: npm run build
       - run: npm test
+      - if: ${{ matrix.node-version != '14.x' }}
```

Comment:
Hopefully we can soon remove 14.x from our matrix entirely but, does it fail currently for 14?

Reply:
Yes; but the error seems unrelated to `esbuild` itself and more to do with something going wrong with other dependencies (see https://github.com/inrupt/universal-fetch/actions/runs/5735387992/job/15543051983).

I don't think it is worth investigating this since Node 14 is already at EOL.